libnavigation-copilot/src/main/java/com/mapbox/navigation/copilot/MapboxCopilotImpl.kt

```diff
@@ -109,15 +110,25 @@ internal class MapboxCopilotImpl(
     }
     private var initRoute = false
     private lateinit var routesObserver: RoutesObserver
+    private val isMapboxNavigationAppSetup = MapboxNavigationApp.isSetup()
+    private var isCopilotInitialized = false
 
     /**
      * start
      */
     fun start() {
+        if (isCopilotInitialized) {
+            return
+        }
+        isCopilotInitialized = true
```

Comment:
I expected this logic to be in `MapboxCopilot`. Looks like it is still an issue to call `MapboxCopilot.onAttached`.

Reply:
Great catch! I've just pushed fixing ☝️